test/script-tests/README

```diff
@@ -1,4 +1,4 @@
-This is a fresh start for script tests. The fact that windows exists can
+This is a fresh start for script tests. The fact that windows exist can
```

Comment:
Are you sure you want to touch this file? I think it means you have to fix the script tests.

"windows" is a singular proper noun for the OS. It would be capitalized in proper English but maybe it doesn't deserve to be.

Reply:
Right, it should be "Windows".

Reply:
"windoze." I can only guess at the end of the sentence: "The fact that windoze still exists can only portend a more horrible end to humanity than has yet been prognosticated." Wait, is this like the one-question FAQ, except it's the one-test windoze script test suite?